tests/test_magic_links.py

```diff
@@ -20,7 +20,7 @@ def test_magic_link_is_created(self, flask_test_client):
         THEN a magic link is created and returned
         :param flask_test_client:
         """
-        expected_link_attributes = {"accountId": "userA"}
+        expected_link_attributes = {"accountId": "usera"}
```

Comment:
💯 Wow!

Reply:
so conventional, he's making up conventions for no apparent reason